Sofa/Component/Engine/Analyze/src/sofa/component/engine/analyze/Distances.inl

```diff
@@ -47,7 +47,7 @@ Distances< DataTypes >::Distances ( sofa::component::topology::container::dynami
     showGradientMap ( initData ( &showGradientMap, false, "showGradients","show gradients for each point of the target point set." ) ),
     showGradientsScaleFactor ( initData ( &showGradientsScaleFactor, 0.1, "showGradientsScaleFactor","scale for the gradients displayed." ) ),
     offset ( initData ( &offset, Coord(), "offset","translation offset between the topology and the point set." ) ),
-    distanceType ( initData ( &distanceType, TYPE_GEODESIC, "distanceType","type of distance to compute for inserted frames." ) ),
+    distanceType ( initData ( &distanceType, {"Geodesic","Harmonic","Stiffness Diffusion", "Vorono\xEF", "Harmonic with Stiffness"}, "distanceType","type of distance to compute for inserted frames." ) ),
```

Comment:
```suggestion
    distanceType ( initData ( &distanceType, {"Geodesic","Harmonic","Stiffness Diffusion", "Voronoi", "Harmonic with Stiffness"}, "distanceType","type of distance to compute for inserted frames." ) ),
```
I dont know if we support UTF-8 😅 and I dont think anybody will type "Vorono\xEF" in their scene.
Should we just put a normal `i` ?